target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,24 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-wr650ac-v1
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-WR650AC
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-i2c-gpio
```

Comment:
No i2c on this board, so the `kmod-i2c-gpio` package should be dropped.

Reply:
done